docs/02.deploying/03.rancher/03.rancher.md

```diff
@@ -74,6 +74,142 @@ Note in the above screen shot, two Rancher users admin and gkosaka have been aut
 :::
 
+#### NeuVector/Rancher SSO Permission Resources
+
+The Rancher v2.9.2 UI provides for selecting NeuVector permission resources when creating `Global/Cluster/Project/Namespaces` roles. When a Rancher user is assigned a role with a NeuVector permission resource, the user's NeuVector SSO session is assigned the respective NeuVector permission accordingly. This is to provide SSO users with custom roles other than the reserved `admin/reader/fedAdmin/fedReader` roles.
+
+Below are the mapped permission resources used with applicable `Global/Cluster/Project/Namespaces` roles.
+
+##### Mapped Permission Resources for `Global/Cluster` Role
+
+:::note
+Users will need to manually add * (Verbs) / services/proxy (Resource) to NeuVector-related `Global/Cluster` Roles.
+:::
+
+API Groups:
+
+`permission.neuvector.com`
+
+Verbs:
+
+```shell
+get    // for read-only(i.e. view)
+*      // for read/write(i.e. modify)
+```
+
+Resources:
+
+NeuVector, Cluster Scoped
+
+```shell
+AdmissionControl
+Authentication
+CI Scan
+Cluster
+Federation
+Vulnerability
+```
+
+NeuVector, Namespaced
+  
+```shell
+AuditEvents
+Authorization
+Compliance
+Events
+Namespace
+RegistryScan
+RuntimePolicy
+RuntimeScan
+SecurityEvents
+SystemConfig
+```
+
+Resource display for Rancher `Global/Cluster` Role Template pages:
+
+apiGroup for NeuVector
+
+`api.neuvector.com`
+
+```shell
+[resource display]   [resource]
+--------------------------------------------------
+"All Permissions":   nv-perm.all-permissions
+"Admission Control": nv-perm.admctrl
+"Audit Events":      nv-perm.audit-events
+"Authentication":    nv-perm.authentication
+"Authorization":     nv-perm.authorization
+"CI Scan":           nv-perm.ci-scan
+"Compliance":        nv-perm.compliance
+"Events":            nv-perm.events
+"Federation":        nv-perm.fed
+"Registry Scan":     nv-perm.reg-scan
+"Runtime Policy":    nv-perm.rt-policy
+"Runtime Scan":      nv-perm.rt-scan
+"Security Events":   nv-perm.security-events
+"System Config":     nv-perm.config
+"Vulnerability Profile":     nv-perm.vulnerability
+```
+
+##### Mapped Permission Resources for `Project/Namespace` Role
+
+:::note
+Users will need to manually add * (Verbs) / services/proxy (Resource) to NeuVector-related `Project/Namespace` Roles.
+:::
+
+API Groups:
+
+`permission.neuvector.com`
+
+Verbs:
+
+```shell
+get    // for read-only(i.e. view)
+*      // for read/write(i.e. modify)
+```
+
+Resources:
+
+NeuVector, Namespaced
+  
+```shell
+AuditEvents
+Authorization
+Compliance
+Events
+Namespace
+RegistryScan
+RuntimePolicy
+RuntimeScan
+SecurityEvents
+SystemConfig
+```
+
+Resource display for Rancher `Project` Role Template pages:
```

Comment:
Please remove lines 188~211
They are not available in Rancher's role creation -> resource drop-down menu